lib/python/qmk/cli/new/keymap.py

```diff
@@ -0,0 +1,39 @@
+"""This script automates the copying of the default keymap into your own keymap.
+"""
+import os
+import shutil
+
+from milc import cli
+
+
+@cli.argument('-k', '--keyboard', help='Specify keyboard name. Example: 1upkeyboards/1up60hse')
+@cli.argument('-u', '--username', help='Specify any name for the new keymap directory')
+@cli.entrypoint('Create a new keymap the keyboard of your choosing')
+def main(cli):
+    # ask for user input if keyboard or username was not provided in the command line
```

Comment:
@skullydazed I may have missed your comment about saying "Collect information the user didn't provide" when I was resolving comments from before. lmk if you still feel I can reword what I got here to be closer to what you suggested earlier for more clarity.

Reply:
You have the correct behavior here, I'm fine with this.

You probably want to put a docstring on main(), however, because pydoc will interpret the comment as documentation.